styles/Booklist.module.css

```diff
@@ -0,0 +1,54 @@
+.card {

```

Comment:
To confirm - you are writing your CSS by hand? You're not using any css frameworks like bootstrap, tailwind, etc.?

Reply:
Yes sir by hand. I plan to work on the css frameworks after I finished all the functionalities.